tests/ssg_test_suite/test_env.py

```diff
@@ -219,17 +219,32 @@ def __init__(self, mode, hypervisor, domain_name):
 
         self._origin = None
 
+    def snapshot_lookup(self, snapshot_name):
+        try:
+            snapshot = self.domain.snapshotLookupByName(snapshot_name)
+        except Exception as exc:
+            raise RuntimeError(exc)
+        return snapshot
+
+    def snapshot_cleanup(self):
```

Comment:
I think there should be some kind of question asking the user if he is really sure about deleting the snapshot. Possibly with an option to force the removal or to automatically answering `yes` to the question.

Reply:
I particularly wound't like to introduce a question since this would break the automation and by demanding user interaction. But the inclusion of an option could be a good alternative. This would demand an explicit extra option that seems unnecessary, but maybe someone would like to keep old ssgts snapshots for some reason. I will include this since it is better to have if necessary than not having it. : )

Reply:
I believe you can make the default behavior to delete the snapshots and add an option that would not force the removal of the snapshots, this way it would keep the old behavior for people that don't want to force remove them.